packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -771,12 +771,16 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   }
 
   void _handleDragEnd(DragEndDetails details) {
-    _backGestureController?.dragEnd(details.velocity.pixelsPerSecond.dx / context.size.width);
+    bool willPop = _backGestureController?.dragEnd(details.velocity.pixelsPerSecond.dx / context.size.width);
+    if (willPop == true)
```

Comment:
Rather than explicitly testing against `true` here, I'd put a `??` on line 774 to always store a non-null value in `willPop`.  It just makes it less likely someone will refactor this code in the future and remove the null handling.